frontend/app/form/jobdescriptionForm.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import { useState } from "react";
+import {
+	Button,
+	Card,
+	CardContent,
+	CardHeader,
+	TextField,
+} from "@mui/material";
+
+export default function JobDescriptionForm() {
+	const [description, setDescription] = useState<string>("");
+	const [message, setMessage] = useState<string | null>(null);
+
+	/**
+	 * Requires task 8 to submit form data
+	 * For now, I created a mock to submit and
+	 * return a response
+	 */
+	const uploadJobDescription = async (
+		_description: string,
+	): Promise<string> => {
+		return Promise.resolve("Job description submitted successfully.");
+	};
+
+	const handleSubmit = async (event: React.FormEvent) => {
+		event.preventDefault();
+
+		// Check if the job description is empty
+		if (!description) {
+			setMessage("Job description cannot be empty.");
+			return;
+		}
+
+		// Check if it char count exceeds 5000
+		if (description.length > 5000) {
+			setMessage("Job description must be under 5000 characters.");
+			return;
+		}
+
+		// Submits job description
+		try {
```

Comment:
use something like
```js
return uploadJobDescription().then().catch()
```
instead of try/catch

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch

Reply:
might want to read up on promises and async functions if youre unfamiliar with them:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#description